packages/js/src/executors/node/node.impl.ts

```diff
@@ -36,6 +36,7 @@ export async function* nodeExecutor(
   options: NodeExecutorOptions,
   context: ExecutorContext
 ) {
+  process.env.NODE_ENV ??= context?.configurationName ?? 'development';
```

Comment:
Should these both default to the same value? Currently esbuild defaults to production and this hits dev

Reply:
well this is serve we don't serve 'production' by default IIRC